packages/@react-types/actiongroup/src/index.d.ts

```diff
@@ -35,7 +35,12 @@ export interface ActionGroupProps<T> extends MultipleSelection {
    * Invoked when an action is taken on a child. Especially useful when `selectionMode` is none.
    * The sole argument `key` is the key for the item.
    */
-  onAction?: (key: Key) => void
+  onAction?: (key: Key) => void,
+  /**
+   * Whether the ActionGroup is displayed in a Toolbar.
+   * @default false
+   */
+  isInsideAToolbar?: boolean
```

Comment:
need a better name for this

Reply:
how about `isInToolbar`? pretty much the same but just slightly more concise